http-netty/src/main/java/io/micronaut/http/netty/channel/DefaultEventLoopGroupRegistry.java

```diff
@@ -41,25 +43,31 @@
 @BootstrapContextCompatible
 public class DefaultEventLoopGroupRegistry implements EventLoopGroupRegistry {
     private final EventLoopGroupFactory eventLoopGroupFactory;
-    private final ThreadFactory threadFactory;
     private final BeanLocator beanLocator;
 
     /**
      * Default constructor.
      *
      * @param eventLoopGroupFactory The event loop group factory
-     * @param threadFactory         The thread factory
      * @param beanLocator           The bean locator
      */
-    public DefaultEventLoopGroupRegistry(
-            EventLoopGroupFactory eventLoopGroupFactory,
-            @Named(NettyThreadFactory.NAME) ThreadFactory threadFactory,
-            BeanLocator beanLocator) {
+    public DefaultEventLoopGroupRegistry(EventLoopGroupFactory eventLoopGroupFactory, BeanLocator beanLocator) {
```

Comment:
If you do as I suggested with using `@Parameter` then we don't need to change this constructor which is a breaking change. Otherwise we will need a new constructor and to annotate the new one with `@Inject` and deprecate the old to not break code

Reply:
The class is annotated as `@Internal` so I assume no one should be using it.

Reply:
Ah ok